notebooks/summit/AuxTel/SITCOM-476/SITCOM-476_LargeDonuts.ipynb

```diff
@@ -0,0 +1,761 @@
+{
```

Comment:
<p>might be worth adding a link to the ticket.</p><p>In fact, I wonder if each notebook should have a link to the ticket in the title section just as a standard thing...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
👍 I usually add the link to the jira ticket in the title section.